packages/toml/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "prettier-plugin-toml",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "An opinionated `toml` formatter plugin for Prettier",
+  "repository": "git@github.com/un-ts/prettier.git",
+  "homepage": "https://github.com/un-ts/prettier/tree/master/packages/prettier",
+  "author": "JounQin (https://www.1stG.me) <admin@1stg.me>",
+  "funding": "https://opencollective.com/unts",
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "main": "./lib/index.cjs",
+  "module": "./lib/index.js",
+  "exports": {
+    "types": "./lib/index.d.ts",
+    "import": "./lib/index.js",
+    "require": "./lib/index.cjs"
+  },
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "keywords": [
+    "toml",
+    "plugin",
+    "prettier",
+    "prettier-plugin"
+  ],
+  "peerDependencies": {
+    "prettier": "^3.0.3"
+  },
+  "dependencies": {
+    "@taplo/lib": "0.4.0-alpha.2"
```

```suggestion
    "@taplo/lib": "^0.4.0-alpha.2"
```

I'd prefer to use `^` version.